### PR TITLE
Addition of ADXCVR status bits for more specific check

### PR DESCRIPTION
**Pull request by danmois**
The checking of status bits [5] and [6] introduced in the ADXCVR HDL and the synchronization with the Linux driver for staus check.

### CODE REVIEW

File: drivers/axi_core/jesd204/axi_adxcvr.c [hunk 1]
```diff
@@ -56,9 +56,12 @@
 /******************************************************************************/
 #define ADXCVR_REG_RESETN		0x0010
 #define ADXCVR_RESETN			(1 << 0)
+#define ADXCVR_BUFSTATUS_RST		(1 << 1)
```
Comment (CiprianRegus) [inline drivers/axi_core/jesd204/axi_adxcvr.c:59]: `NO_OS_BIT(1) ` ?

File: drivers/axi_core/jesd204/axi_adxcvr.c [hunk 2]
```diff
@@ -56,9 +56,12 @@
...
 
-#define ADXCVR_REG_STATUS		0x0014
-#define ADXCVR_STATUS			(1 << 0)
```
Comment (amiclaus) [inline drivers/axi_core/jesd204/axi_adxcvr.c:61]: these changes are unrelated.

Comment (danmois) [reply drivers/axi_core/jesd204/axi_adxcvr.c:61]: Removed modification.

**Comment by danmois** [comment]
V3

- removed unrelated changes
- use of NO_OS_BIT macro for defining ADXCVR_BUFSTATUS_RST

**Review by amiclaus — APPROVED** [review]
